config/locales/es.yml

```diff
@@ -34,30 +34,30 @@ es:
       authentication:
         disabled: Deshabilitada
         enabled: Habilitada
-        info: Permitir solo a usuarios autenticados ingresar a una sala
+        info: Permitir solo a usuarios autenticados acceder a una sala
```

Comment:
acceder/ingresar in this context ids the same meaning

Reply:
Este cambio es solamente por consistencia.